.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -61,7 +65,29 @@ steps:
     env:
       MODE: "windowsVhdMode"
       SUBSCRIPTION_ID: $(SUBSCRIPTION_ID)
+  - script: |
+      mkdir -p $(windows.cse.working.dir)
```

Comment:
We should put this in a seperate .sh file - that way shellcheck will check the scripts for potential problems. Shellcheckignores scripts embedded in the pipeline yaml.